### PR TITLE
Fix #1182

### LINKED ISSUE #1182: Zero results after signing in with FxA and email case doesn't match
So, apparently I appear in 13 breaches if I don't sign in, but if I sign in, my same email appears in no breaches at all. Not a very good system at monitoring if the same exact email signed in isn't being detected properly. And yes, I literally mean the same exact. I logged out and checked the same email copy and pasted and it shown up in 13. So signing in shows zero breaches while signing out shows all.

image

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Lower cases new emails before adding to DB.
-  Updates `bundleVerifiedEmails` to convert existing user emails to lowercase letters before making a new sha1 hash (of the newly lowercased email) to use for scanning breaches. 

Screen Shot 2019-09-04 at 5 58 25 PM
Screen Shot 2019-09-04 at 5 58 31 PM

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -136,7 +136,8 @@ async function add(req, res) {
 }
 
 async function bundleVerifiedEmails(email, emailSha1, ifPrimary, id, verificationStatus, allBreaches) {
-  const foundBreaches = await HIBP.getBreachesForEmail(emailSha1, allBreaches, true);
+  const lowerCaseEmailSha = sha1(email.toLowerCase());
```
Comment (pdehaan) [inline controllers/user.js]: Is `emailSha1` now unused?

**Comment by pdehaan** [comment]
> * Lower cases new emails before adding to DB.

Do we need to update the existing records in the DB and lowercase some emails/SHAs?

File: controllers/user.js (hunk 1 above)
Comment (lesleyjanenorton) [reply controllers/user.js]: indeed it is!

**Comment by lesleyjanenorton** [comment]
@pdehaan re: your last comment, I would think that updating existing records would be ideal right? @groovecoder what would that look like/is it feasible? 

Also curious: I'm no longer able to add variations of `lesleyjanenorton@gmail.com` without triggering the duplicate email error.

**Pull request closed without merging**